src/pages/components/table-of-contents.mdx

```diff
@@ -32,7 +32,16 @@ The Table of Contents (ToC) component renders as left navigation that includes a
 
 <Title>Use case</Title>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+![Image of Table of contents component usecase](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+
+**When to use:**
+* When the content of the page is long or text heavy and will require a lot of scrolling.
+* When users need to know what is included in a page, the table of contents can serve as an outline or summary of the page content.
+* When users need to navigate to a specific section on the page quickly.
+
+**When not to use:**
+* When a page is relatively short and the user will not have to scroll very far.
+* When users need to read all the content in a specific order, consider that the table of contents allows users to skip content easily.
```

Comment:
> consider that the table of contents allows users to skip content easily.

This line reads awkward to me... What are we trying to say?

Reply:
I suggest we have it as "When users need to read all the content in a specific order."